src/components/Tabs/Tabs.less

```diff
@@ -12,53 +14,112 @@
 	}
 
 	&-Tab {
-		color: @color-primary;
 		flex: 1 1 auto;
 		cursor: pointer;
+		color: @color-primary;
 		user-select: none;
-		margin-bottom: -1px;
+		margin-bottom: -@Tabs-border-width;
 		height: @Tabs-tab-height;
 		display: flex;
 		align-items: center;
 		justify-content: center;
-		padding-top: @Tabs-spacer-height;
-		border: 1px solid @color-default-borderColor;
+		border: @Tabs-border-width solid @color-default-borderColor;
 		border-right: 0;
 		background-color: @color-white;
 
 		&:last-of-type {
-			border-right: 1px solid @color-default-borderColor;
+			border-right: @Tabs-border-width solid @color-default-borderColor;
 		}
 
 		&:hover {
-			background-color: @color-gray-5;
+			background-color: @color-lightGray;
+			.Tabs-mixin-tab-arrow-background(@color-lightGray);
 		}
 
 		&:active {
-			box-shadow: @shadow-inset;
+			background-image: linear-gradient(@color-gray 0, @color-lightGray @Tabs-spacer-height);
```

Comment:
just a note to switch this to the new gradient mixin when Vertical List is merged